src/Pecee/Http/Input/ValidatorRules/ValidatorRuleMax.php

```diff
@@ -2,28 +2,46 @@
 
 namespace Pecee\Http\Input\ValidatorRules;
 
+use Pecee\Http\Input\Exceptions\InputValidationException;
 use Pecee\Http\Input\IInputItem;
+use Pecee\Http\Input\InputFile;
 use Pecee\Http\Input\InputValidatorRule;
 
-class ValidatorRuleMax extends InputValidatorRule{
+class ValidatorRuleMax extends InputValidatorRule
+{
 
     protected $tag = 'max';
 
-    public function getMax(): int{
-        if(sizeof($this->getAttributes()) > 0){
-            return intval($this->getAttributes()[0]);
+    public function getMax(): int
```

Comment:
Watch out for the return type. We have to remove it in this PHP version, because we can't do int|float.